analyzer/src/test/kotlin/managers/utils/NodeSupportTest.kt

```diff
@@ -272,6 +283,12 @@ class NodeSupportTest : WordSpec() {
 
     private fun absolutePaths(vararg files: String) = files.map { tempDir.resolve(it) }
 
+    private fun containAbsolutePathsExactly(vararg paths: String): Matcher<Collection<File>?> =
+        containExactly(absolutePaths(*paths) as Collection<File>)
+
+    private fun containAbsolutePathsExactlyInAnyOrder(vararg paths: String): Matcher<Collection<File>?> =
+        containExactlyInAnyOrder(absolutePaths(*paths))
+
```

Comment:
Hmm. I have do admit I do not like this change. It doesn't really increase readability significantly, at the cost of introducing very specialized functions. How maybe other permutations of `shouldContainExactly(InAnyOrder) <helper function>` do we want to wrap in yet another helper function.

I'd prefer to drop this commit.

Reply:
Removed it.

Reply:
Thanks.